packages/vuetify/src/composables/border-radius.ts

```diff
@@ -1,15 +1,18 @@
 // Utilities
 import { computed } from 'vue'
+import type { Prop } from 'vue'
 import propsFactory from '@/util/propsFactory'
 
 // Types
+export type IRounded = typeof radius[number]
```

Comment:
This line is based on the const a few lines below, that export type would be best AFTER not before, like it is at the moment.